MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1130,6 +1130,20 @@ public string GetVideoQualityParam(EncodingJobInfo state, string videoEncoder, E
             var profile = state.GetRequestedProfiles(targetVideoCodec).FirstOrDefault();
             profile =  Regex.Replace(profile, @"\s+", String.Empty);
 
+            // We only transcode to HEVC 8-bit for now, force Main Profile.
+            if (profile != null
+                && (profile.IndexOf("main 10", StringComparison.OrdinalIgnoreCase) != -1
+                    || profile.IndexOf("main still", StringComparison.OrdinalIgnoreCase) != -1))
```

Comment:
```suggestion
            if (profile.Contains("main 10", StringComparison.OrdinalIgnoreCase)
                    || profile.Contains("main still", StringComparison.OrdinalIgnoreCase))
```

`profile` is known to be not null here.
`Contains` now has a `StringComparison` parameter so we don't need to use `IndexOf` anymore